web/src/components/hooks/useFeatures.ts

```diff
@@ -1,8 +1,13 @@
 import * as React from 'react';
 import { useURLPoll } from './useURLPoll';
+import { cancellableFetch } from '../cancellable-fetch';
 
 const URL_POLL_DEFAULT_DELAY = 60000; // 60 seconds
 
+export enum POLL_DELAY {
+  oneMinute = URL_POLL_DEFAULT_DELAY,
+}
+
```

Comment:
Does this need to be an enum with a single value?

Reply:
I removed the enum in the latest push.